### PR TITLE
B/update guzzle

### PR DESCRIPTION
**Pull request by bogensberger**

### CODE REVIEW

File: test/CrateTest/PDO/Http/ServerTest.php [hunk 1]
```diff
@@ -11,6 +11,7 @@
 use Crate\PDO\Exception\UnsupportedException;
 use Crate\PDO\Http\Server;
 use GuzzleHttp\ClientInterface as HttpClientInterface;
```
Comment (kovrus) [inline test/CrateTest/PDO/Http/ServerTest.php]: not used anymore

File: src/Crate/PDO/Http/Server.php [hunk 2]
```diff
@@ -38,29 +38,35 @@ class Server implements InternalClientInterface
...
+    ];
+
     /**
      * @param string $uri
      * @param array  $options
      */
     public function __construct($uri, array $options)
     {
         $uri = self::computeURI($uri);
-        $this->client = new HttpClient(['base_url' => $uri] + $options);
```
Comment (chaudum) [inline src/Crate/PDO/Http/Server.php]: `$options` are lost?

**Comment by chaudum** [comment]
changelog entry is missing

**Comment by chaudum** [comment]
otherwise lgtm

**Pull request merged**